packages/web/components/Player/Section/PlayerHero.tsx

```diff
@@ -19,67 +20,49 @@ import {
   Wrap,
   WrapItem,
 } from '@metafam/ds';
+import { Maybe } from '@metafam/utils';
 import BackgroundImage from 'assets/main-background.jpg';
 import { FlexContainer } from 'components/Container';
 import { EditProfileForm } from 'components/EditProfileForm';
 import { PlayerAvatar } from 'components/Player/PlayerAvatar';
-import { PlayerContacts } from 'components/Player/PlayerContacts';
+import { PlayerContacts as Contacts } from 'components/Player/PlayerContacts';
+import { PlayerBrightId as BrightId } from 'components/Player/Section/PlayerBrightId';
```

Comment:
PlayerBrightId was removed from PlayerHero and made exclusive to `PlayerContacts`. Could we revert this to how it was? 

This is because we do not want to see "Verify On BrightId" right below the username and also its not working. So it should be disabled as it is in PlayerContacts.

Reply:
I'll add it to [the list](//github.com/MetaFam/TheGame/issues/1146#issuecomment-1053911232).